### PR TITLE
Use below/above language in the alert form. Add default boundaries file

### PR DESCRIPTION
**Pull request by ericboucher**
- Re-order above / below thresholds
- transform missing min/max to undefined before sending the alert to the API
- Add "true" default to alert `active` field

### CODE REVIEW

**Review by Ry-DS — APPROVED** [review]
Looks good :+1:

File: src/context/analysisResultStateSlice.ts [hunk 1]
```diff
@@ -61,6 +61,10 @@ function getAdminBoundariesURL() {
   if (adminBoundariesPath.startsWith('http')) {
     return adminBoundariesPath;
   }
+  // do not send a local path to the API, use a fixed boundary file instead.
+  if (window.location.origin.includes('localhost')) {
```
Comment (Ry-DS) [inline src/context/analysisResultStateSlice.ts:65]: For this, would it make more sense to check the environment and see if its in development mode? And then refer to this url as a constant? Trying to think of ways to make it consistent to how we load development mode in general

Comment (Ry-DS) [reply src/context/analysisResultStateSlice.ts:65]: maybe in src/config/index.ts somewhere

Comment (ericboucher) [reply src/context/analysisResultStateSlice.ts:65]: This breaks if the country changes but the window includes localhost is a safe test that we are in development mode, no?

**Pull request merged**